web/src/components/SocialMediaTooltip.tsx

```diff
@@ -0,0 +1,236 @@
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+
+    ${props =>
+      props.flow === 'top' &&
+      css`
+        left: 20px;
+        bottom: -8px;
+        transform: rotate(-180deg);
+      `};
+    ${props =>
+      props.flow === 'right' &&
+      css`
+        left: -14px;
+        transform: rotate(-90deg);
+        top: 45%;
+      `};
+    ${props =>
+      props.active &&
+      css`
+        animation: tooltips 300ms ease-out forwards;
+      `};
+  }
+
+  &:after {
+    z-index: 1000;
+    border-bottom: 11px solid ${props => props.theme.colors.textDecorationColor};
+    border-left: 11px solid transparent;
+    border-right: 11px solid transparent;
+
+    ${props =>
+      props.flow === 'top' &&
+      css`
+        left: 20px;
+        bottom: -11px;
+        transform: rotate(-180deg);
+      `};
+    ${props =>
+      props.flow === 'right' &&
+      css`
+        left: -17px;
+        transform: rotate(-90deg);
+        top: 45%;
+      `};
+    ${props =>
+      props.active &&
+      css`
+        animation: tooltips 300ms ease-out forwards;
+      `};
+  }
+
+  @keyframes tooltips {
+    to {
+      opacity: 1;
+    }
+  }
+`
+
+const CloseButton = styled.button`
+  background-color: ${props => props.theme.colors.backgroundColor};
+  border: none;
+  padding: 0;
+  display: flex;
+`
+
+const Link = styled.a`
+  background-color: ${props => props.theme.colors.backgroundColor};
+  border: none;
+  padding: 0;
+  display: flex;
+`
+
+const Icon = styled.img<{ direction: string }>`
+  width: 32px;
+  height: 32px;
+  flex-shrink: 0;
+  padding: 8px;
+  object-fit: contain;
+  align-self: center;
+
+  ${props =>
+    props.direction === 'rtl' &&
+    css`
+      transform: scaleX(-1);
+    `};
+`
+
+const BackdropContainer = styled.div`
+  background: transparent;
+  width: 100%;
+  height: 100%;
+  z-index: 2000;
+  top: 0;
+  left: 0;
+  position: fixed;
+`
+
+const SocialMediaTooltip = ({
+  active,
+  children,
+  shareLink,
+  onClose,
+  title,
+  flow,
+  direction,
+  route,
+}: SocialMediaTooltipProps): ReactElement => {
+  const { t } = useTranslation('socialMedia')
+  const { viewportSmall } = useWindowDimensions()
+  const isPoisDetailPage = route === POIS_ROUTE && getSlugFromPath(window.location.pathname) !== POIS_ROUTE
+  const encodedTitle = encodeURIComponent(title)
+  const encodedShareLink = encodeURIComponent(shareLink)
+
+  return (
+    <div>
+      {active && (
+        <>
+          <BackdropContainer onClick={onClose} role='button' tabIndex={0} onKeyPress={onClose} />
+          <TooltipContainer
+            flow={flow}
+            active={active}
+            direction={direction}
+            additionalPadding={isPoisDetailPage && !viewportSmall ? dimensions.poiDetailNavigation : 0}>
+            <Tooltip text={t('whatsappTooltip')} flow='up'>
+              <Link
+                href={`https://api.whatsapp.com/send?text=${encodedTitle}%0a${encodedShareLink}`}
```

Comment:
I think we should include the same share message as on native (`Gerade gefunden: `).

Reply:
nice find